### PR TITLE
Fix presets

### PR DESCRIPTION
**Pull request by haslinghuis**
Found intermittent issue when using load backup in presets tab.

Fixes:

image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by limonspb — APPROVED** [review]

**Comment by limonspb** [comment]
interesting finding... i guess the tab is being destroyed? sometimes... I don't understand when THIS becomes invalid.

**Review by McGiverGim — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
